impress-org/givewp
# Changed: Update toggle component and filter attribute

Resolves #6483

## Description

This PR updates the filter attribute name and passes a new set of options to be  used within the Toggle component for the Donor Wall. 

## Affects

Donor Wall
Toggle block component

## Visuals


https://user-images.githubusercontent.com/75056371/175352455-24c131ce-8493-4d7d-a3e3-45c02e3a29d2.mov


## Testing Instructions

- Use the Donor Wall block or shortcode
- Select the block and open the settings for "Display Elements"
- Toggle between both options to verify that the selected options background color changes & that the list of toggles change
- Open the "Wall Settings" and use the filterable options to verify they continue to work as expected.
- 
## Pre-review Checklist
